jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
As a result of json_group_array(), list in result.data is of type unicode. Not sure how to convert it to list in sql.

```
>>> result.data[0][0]
u'["a","b","c","d","e","f","g","h","i","j","x","y","z"]'


>>> type(result.data[0][0])
<type 'unicode'>Inline images 1
```